test/commons/individual.test.coffee

```diff
@@ -0,0 +1,45 @@
+require("../test-suite")()
+Individual = require('../../src/model/individual')
+
+describe 'Individual', ->
+
+  payload_ok = null
+  payload_ko = null
+
+  beforeEach ->
+    payload_ok = {
+        type: "$INDIVIDUAL",
+        id: "ciptr4z1f00093k6krpx3vuhe",
+        attributes: {
+            name: "Unnamed",
+            type: "$INDIVIDUAL"
+        },
+        relations: { }
+      }
+
+    payload_ko = {
+        type: "$INDIVIDUAL",
+        attributes: {
+            name: "Unnamed",
+            type: "$INDIVIDUAL"
+        },
+        relations: { }
+      }
+
+
+  it 'should create a instance of $INDIVIDUAL', (done) ->
+    individual = new Individual(payload_ok)
+    if individual.isValid()
+      console.log "       Individual id: " + individual.id
+      done()
+
+    else
+      console.log 'error'
+
```

Comment:
Don't use the done() callback as a way of providing feedback about the test. Instead use expects or asserts. 

So either
```
  it 'should create a instance of $INDIVIDUAL', ->
     individual = new Individual(payload_ok)
     if individual.isValid()
       console.log "       Individual id: " + individual.id
     else
       assert.fail("valid", "invalid")
```
or even  more compact:
```
  it 'should create a instance of $INDIVIDUAL', ->
     individual = new Individual(payload_ok) 
     expect(individual.isValid()).to.equal(true)
```